src/bootstrap/lib/overload-protection.js

```diff
@@ -0,0 +1,32 @@
+const configure = ({
```

Comment:
This matches the existing pattern of abstracting functions away into libs so they can be throughly tested